diff --git i/libgalois/include/galois/substrate/NumaMem.h w/libgalois/include/galois/substrate/NumaMem.h
index c9cbe6f2c..97598426c 100644
--- i/libgalois/include/galois/substrate/NumaMem.h
+++ w/libgalois/include/galois/substrate/NumaMem.h
@@ -20,7 +20,7 @@
 #ifndef GALOIS_SUBSTRATE_NUMAMEM
 #define GALOIS_SUBSTRATE_NUMAMEM
 
-#include <cstddef>
+#include <cstdint>
 #include <memory>
 #include <vector>
 
diff --git i/libgalois/src/HWTopoLinux.cpp w/libgalois/src/HWTopoLinux.cpp
index 0835b0070..c34d2db8a 100644
--- i/libgalois/src/HWTopoLinux.cpp
+++ w/libgalois/src/HWTopoLinux.cpp
@@ -26,12 +26,14 @@
 #include <array>
 #include <cassert>
 #include <cerrno>
+#include <cstdio>
 #include <cstring>
 #include <fstream>
 #include <functional>
 #include <memory>
 #include <mutex>
 #include <set>
+#include <string>
 
 #ifdef GALOIS_USE_NUMA
 #include <numa.h>
@@ -101,7 +103,7 @@ std::vector<cpuinfo> parseCPUInfo() {
   std::vector<cpuinfo> vals;
 
   const int len = 1024;
-  std::array<char, len> line;
+  std::string line;
 
   std::ifstream procInfo("/proc/cpuinfo");
   if (!procInfo)
@@ -109,8 +111,7 @@ std::vector<cpuinfo> parseCPUInfo() {
 
   int cur = -1;
 
-  while (true) {
-    procInfo.getline(line.data(), len);
+  while (std::getline(procInfo, line)) {
     if (!procInfo)
       break;
 
